src/CommandHistory/CommandHistoryContext.tsx

```diff
@@ -6,17 +6,15 @@ export const STATUS = {
 } as const;
 type Status = (typeof STATUS)[keyof typeof STATUS];
 
-export interface Command<
-	T extends Record<string, unknown> = Record<string, unknown>,
-> {
+export interface Command<T extends object = Record<string, unknown>> {
```

Comment:
Object isn't a valid type here because all property information is abstracted away and you can't reference any property on an object type variable

Reply:
![image](https://github.com/user-attachments/assets/5f72ffe6-e5be-481c-af9b-e46b882e961f)

Reply:
But the default type is `Record` which will allow accessing any key and if you want a more precise type you can pass it there only if extends `object`.

Reply:
As I have done in command added in this pr.